src/main/java/sparksoniq/spark/iterator/SparkRuntimeIterator.java

```diff
@@ -65,6 +66,7 @@ public Item next(){
             this._rdd = this.getRDD();
             if(SparkContextManager.LIMIT_COLLECT()) {
                 result = _rdd.take(SparkContextManager.COLLECT_ITEM_LIMIT);
+                ShellStart.terminal.output("\nWarning: Results have been truncated to: " + SparkContextManager.COLLECT_ITEM_LIMIT + " items. This value can be configured with the --result-size parameter at startup.\n");
```

Comment:
The warning should only be printed if the limit is reached, meaning, we should surround this output call with a condition on the size of the result list, and only output if the size of the result list is equal to COLLECT_ITEM_LIMIT.